dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/permission/ResourcePermissionCheckServiceImpl.java

```diff
@@ -371,13 +370,17 @@ public List<AuthorizationType> authorizationTypes() {
 
         @Override
         public boolean permissionCheck(int userId, String url, Logger logger) {
-            return false;
+            return true;
```

Comment:
PTAL @WangJPLeo

Reply:
I find we return false in other methods in `ResourcePermissionCheckServiceImpl`, we may need to check this.

Reply:
This is the permission control to control the WorkerGroup DML operation. Currently, other users except for Admin have no permission by default, so this should not be changed to true.